sycl/unittests/Extensions/CommandGraph/Exceptions.cpp

```diff
@@ -571,3 +571,25 @@ TEST_F(CommandGraphTest, ProfilingExceptionProperty) {
   }
   ASSERT_EQ(Success, false);
 }
+
+TEST_F(CommandGraphTest, ClusterLaunchException) {
+  namespace syclex = sycl::ext::oneapi::experimental;
+
+  syclex::properties cluster_launch_property{
+      syclex::cuda::cluster_size(sycl::range<1>{4})};
+
+  std::error_code ExceptionCode = make_error_code(sycl::errc::success);
+  try {
+    Graph.begin_recording(Queue);
+    auto Event1 = Queue.submit([&](sycl::handler &cgh) {
+      cgh.parallel_for<TestKernel<>>(sycl::nd_range<1>({4096}, {32}),
+                                     cluster_launch_property,
+                                     [&](sycl::nd_item<1> it) {});
+    });
+    Queue.wait_and_throw();
```

Comment:
We shouldn't need this call - it could throw an error that will be caught and incorrectly cause the test to pass.

Reply:
Ta